From 32b61c1e56b318c3d9450315e8a016c672ff497a Mon Sep 17 00:00:00 2001
From: Timo Lindfors <timo.lindfors@iki.fi>
Date: Tue, 9 Jul 2013 17:12:21 -0700
Subject: [PATCH 07/31] uhd: fix GRC wrapper generation to match UHD change
 (fixes #562)

From: Timo Juhani Lindfors <timo.lindfors@iki.fi>
Date: Tue, 9 Jul 2013 09:23:06 +0300
Subject: [PATCH] uhd: "scalar" stream arg is now called "peak"

uhd commit d46c176af34b728fd43b3dd46485b38623a7335e (dsp rework: tx
trailer, scaling work (peak)) renamed "scalar" stream arg to "peak"
and changed its semantics. Using 1/256=.003906 captures upper 8-bits.
---
 gr-uhd/grc/gen_uhd_usrp_blocks.py |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gr-uhd/grc/gen_uhd_usrp_blocks.py b/gr-uhd/grc/gen_uhd_usrp_blocks.py
index e392f6a..b7efa07 100644
--- a/gr-uhd/grc/gen_uhd_usrp_blocks.py
+++ b/gr-uhd/grc/gen_uhd_usrp_blocks.py
@@ -141,8 +141,8 @@ self.\$(id).set_bandwidth(\$bw$(n), $n)
 			\#end if
 		</hide>
 		<option>
-			<name>scalar=1024</name>
-			<key>scalar=1024</key>
+			<name>peak=0.003906</name>
+			<key>peak=0.003906</key>
 		</option>
 	</param>
 	<param>
-- 
1.7.10.4

